src/internal.c

```diff
@@ -12382,7 +12382,8 @@ int ProcessPeerCerts(WOLFSSL* ssl, byte* input, word32* inOutIdx,
             }
 
         #if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)
-            if (args->untrustedDepth > ssl->options.verifyDepth) {
+            /* limit compliant with OpenSSL verify Depth + 1 */
```

Comment:
Please expand this comment to explain why "+1" is needed.

Reply:
Agree. Added more context to the comments